update_chroot

```diff
@@ -87,6 +87,40 @@ location = ${COREOS_OVERLAY}
 location = ${PORTAGE_STABLE_OVERLAY}
 EOF
 
+# Automatically symlink any provided portage repo configurations into
+# the appropriate destination directory from "config/portage/repos".
+# CAUTON: Any repo configuration files with the same name, will be silently
+# relinked to the last file found.
+# Automatic file renaming to avoid this is challenging as it has to deal with
+# the case where we actually do want the files overwritten and so is outside
+# the scope of this function for now.
+# If your taking advantage of this functionality in the build toolchain
+# you probably found out about it from these code comments so we can assume
+# for now at least, that you have been somewhat warned.
+
+# If any directories match config/portage/repos/*/repos.conf
```

Comment:
Why bother with the multiple levels of directories? I think looking for `config/portage/repos.conf/*.conf` should be sufficient and has the advantage of mapping directly to `/etc/portage/repos.conf/`.